packages/cli/src/__tests__/plugin.test.js

```diff
@@ -7,17 +17,24 @@ import { getConfig, getPaths } from '@redwoodjs/project-config'
 import * as pluginLib from '../lib/plugin'
 import { loadPlugins } from '../plugin'
 
-jest.mock('fs')
-jest.mock('@redwoodjs/project-config', () => {
+vi.mock('fs-extra', async () => {
+  const memfs = await import('memfs')
   return {
-    getPaths: jest.fn(),
-    getConfig: jest.fn(),
+    default: memfs,
   }
 })
-jest.mock('../lib/packages', () => {
+vi.mock('@redwoodjs/project-config', async (importOriginal) => {
+  const mod = await importOriginal()
```

Comment:
Super minor, but can you please choose a more descriptive name than `mod` here? 
`originalProjectConfig`, `actualProjectConfig`, just `projectConfig`. Something like that maybe? Whatever makes sense to you :)

Reply:
Pushing back a bit because the `mod` name is straight from the Vitest docs here: 

<img width="1532" alt="vitest_docs_mod" src="https://github.com/redwoodjs/redwood/assets/32992335/6ed95d9a-aea9-4aef-8c3a-bb5d85cc92e9">

Taken from the code example here: https://vitest.dev/api/vi.html#vi-mock

Reply:
I think that's because they're mocking `module.js`. Had they mocked `package.js` they'd probably done `const pkg = await importOriginal()`. Don't you think?

Reply:
Not really sure so I tried looking for more examples in Vitest's source. Should they have called this one timers? https://github.com/vitest-dev/vitest/blob/36bc6997c41e6b7aacc05c3d75f391d2bdcd2b77/test/core/test/mock-internals.test.ts#L26

Here they use actual: https://github.com/vitest-dev/vitest/blob/36bc6997c41e6b7aacc05c3d75f391d2bdcd2b77/examples/mocks/test/factory.test.ts#L30

Should this one have been called `fileWasmMod`? https://github.com/vitest-dev/vitest/blob/36bc6997c41e6b7aacc05c3d75f391d2bdcd2b77/test/core/test/imports.test.ts#L110

Reply:
I think they can get away with `mod` because here they're testing their general module importing logic. It has nothing todo with the actual module that's imported.

They do the same thing for regular imports.

Like, we'd never do `const mod = await import('execa')` in our code, right? We always name it something like `const execa = await import('execa')`. But they do that here
![image](https://github.com/redwoodjs/redwood/assets/30793/5f56ae79-9bf2-4e7e-a3a4-a9d3b5ed290e)

Reply:
We could keep going; we tried to stick to the docs to avoid these kinds of discussions around style. I shouldn't have pushed back if I really just wanted an easy review so my bad there; will leave it to you to decide

Reply:
Sorry about the bad tone 🙁  I do value your insight, and I want you to continue to push back whenever you see something you don't agree with!

I'll let you keep using `importOriginal`, just like the docs do. But here I'd like to see a name change to `originalProjectConfig`. And in other places, in general, I want the variable name to indicate what we've imported into it.